spk/firebird/Makefile

```diff
@@ -0,0 +1,33 @@
+SPK_NAME = firebird
+SPK_VERS = 4.0.3
+SPK_ICON = src/firebird.png
+SPK_REV = 1
+
+BETA = 1
+
+DEPENDS = cross/$(SPK_NAME)
+
+include ../../mk/spksrc.common.mk
+
```

Comment:
this can be removed
probably a left over, required only for version checks above th include of spksrc.spk.mk

Reply:
Done.